api/src/main/scala/cryptonite/WalletService.scala

```diff
@@ -0,0 +1,12 @@
+package cryptonite
+
+import scala.concurrent.{Future, ExecutionContext}
+import cryptonite.model.Amount
+import cryptonite.errors.ApiError
+
+class WalletService(repository: WalletRepository)(implicit ec: ExecutionContext) {
+
+  def read(): Future[Either[ApiError, List[Amount]]] = Future {
```

Comment:
@bytecodeguru tua opinione: `ApiError` già nel service ha senso perchè teoricamente i service devono essere pronti per essere esposti direttamente tramite `Wiro`? In GDSM torniamo `String` nel `Left` dei service

Reply:
Non prenderei GDSM come buon esempio in questo caso particolare. Per disaccoppiare errori api per il FE e errori backend, forse si potrebbe  introdurre `ApiError` e `ApplicationError` e fare il mapping tra le due categorie di errori.

Per lo scope di questa PR non so se ha senso fare un refactoring, magari apriamo issue separata (tech debt?) oppure facciamo refactoring nelle prossime PR dopo aver scelto la soluzione definitiva.

Reply:
👍